pkg/util/util.go

```diff
@@ -10,6 +10,11 @@ import (
 	log "github.com/golang/glog"
```

Comment:
Change this line to `"github.com/golang/glog"` could solve the errors in Travis CI

Reply:
Thanks so much, addressed your comments.